Monika-After-Story/MonikaModDev
# farewell menu now adapts to most used fare

The fare menu now has *3* return buttons.

- Goodbye (the generic farewell) is always present on the menu
- The middle option is your most frequently used farewell (as it's less to navigate)
- The final option is `Nevermind` as usual.

I have also made it such that the farewell menu is sorted in alpha-order by prompts.

# Testing:
- Verify fare menu looks good
- ~~Verify this still works on a fresh persistent (Note the order may be kinda random. Do we want to do something for this?)~~
- Verify fares still work
